lib/src/services.dart

```diff
@@ -4,14 +4,18 @@
 
 import 'package:twitch/twitch.dart';
 
-Twitch _twitch;
+class Services {
```

Comment:
For some reason I was running into NPEs with the old setup but I have no idea why -- when I moved the accessors into a Singleton it seemed to work...